app/assets/stylesheets/application.css

```diff
@@ -31,4 +31,10 @@
 
 .home_link {
     color: blue;
-}
\ No newline at end of file
+}
+
+.header {
+    display: flex;
```

Comment:
welcome to flexbox :D 

css was so hard before flexbox, flexbox is god

Reply:
i won't lie, i'm still not 100% sure what flexbox does, but i was told this is how you display things on different sides on the same line lmao

Reply:
haha don't worry for now know it exists and when you need to do more complicated things you can delve a little deeper